Shared/Extensions/URL+BookPlayer.swift

```diff
@@ -12,6 +12,12 @@ public extension URL {
     var isDirectoryFolder: Bool {
         return (try? resourceValues(forKeys: [.isDirectoryKey]))?.isDirectory == true
     }
+    
+    var isInProcessedFolder: Bool {
```

Comment:
I can confirm that this is working, but can we move this to the `DataManager`? by including it in the extension, we're adding a relation to the DataManager from the native URL struct, I would rather have the `DataManager` be the one who owns this check, we could have a class func like this and use that:

```swift
public class func isURLInProcessedFolder(_ url: URL) -> Bool {
    let absoluteUrl = url.resolvingSymlinksInPath().absoluteString
    let processedFolderUrl = getProcessedFolderURL().absoluteString
    return absoluteUrl.contains(processedFolderUrl)
  }
```

Reply:
@GianniCarlo I've moved the code from URL extension to DataManager.